create_package.bat

```diff
@@ -0,0 +1,37 @@
+::
```

Comment:
Change name to windows_create_package.bat

Reply:
why? it fits to create_package.sh for linux.

Reply:
We have all the scripts for windows in main directory named this way. Its easy to execute proper script from command line then and search for it in the ide viewer.

Reply:
ok, that raised to my attention that 2 files should be also renamed to make it clear they are needed only for building - added to PR